### ISSUE TITLE
invalid json in history

### ISSUE THREAD

**Issue by vixns** [vstakhov/rspamd#1812]
### Classification (Please choose *one* option):

* [ ] Crash/Hang/Data loss
* [ ] WebUI/Usability
* [x] Serious bug
* [ ] Other bug
* [ ] Feature
* [ ] Enhancement

### Reproducibility (Please choose *one* option):

* [ ] Always
* [ ] Sometimes
* [ ] Rarely
* [ ] Unable
* [x] I didn’t try
* [ ] Not applicable

### Rspamd version:

1.6.3

### Operation system, CPU, memory and environment:

debian jessie i386

### Description (Please provide a descriptive summary of the issue):

History is stored in redis.
The `FUZZY_DENIED` symbol score is empty.

`"FUZZY_DENIED":{"metric_score":12,"options":["1:fea2662f6e:0.53:txt"],"score":,"name":"FUZZY_DENIED","description":"Denied fuzzy hash"}`

**Comment by WRMSRwasTaken** [comment]
Having the same issue with `BAYES_SPAM` having an empty score causing the invalid JSON.

The log shows: 

```
rspamd[8249]: <ad8192>; task; fuzzy_generate_commands: <5A6C41C4.2040002@bettariniautogru.it>, part is shorter than 1000 bytes: 620 (310 * 2.00 bytes), skip fuzzy check
rspamd[8249]: <ad8192>; task; fuzzy_generate_commands: <5A6C41C4.2040002@bettariniautogru.it>, part is shorter than 1000 bytes: 584 (292 * 2.00 bytes), skip fuzzy check
rspamd[8249]: <ad8192>; surbl; process_dns_results: <5A6C41C4.2040002@bettariniautogru.it> domain [tigercrypt.top.dbl.spamhaus.org] is in surbl dbl.spamhaus.org(601007f)
rspamd[8249]: <ad8192>; surbl; process_dns_results: <5A6C41C4.2040002@bettariniautogru.it> domain [tigercrypt.top.multi.surbl.org] is in surbl multi.surbl.org(40)
rspamd[8249]: <ad8192>; surbl; process_dns_results: <5A6C41C4.2040002@bettariniautogru.it> domain [tigercrypt.top.multi.uribl.com] is in surbl multi.uribl.com(2)
rspamd[8249]: <ad8192>; task; rspamd_task_write_log: id: <5A6C41C4.2040002@bettariniautogru.it>, qid: <4D907F5874>, ip: 93.73.239.187, from: <guido@bettariniautogru.it>, (default: T (reject): [24.39/15.00] [...]
rspamd[8249]: <ad8192>; task; rspamd_protocol_http_reply: regexp statistics: 55 pcre regexps scanned, 8 regexps matched, 163 regexps total, 10 regexps cached, 6.34k bytes scanned using pcre, 6.34k bytes scanned total
rspamd[8247]: <94608b>; proxy; proxy_backend_parse_results: cannot parse input: error while parsing <unknown>: line: 1, column: 131 - 'string value must not be empty', character: ','
rspamd[8247]: <94608b>; proxy; proxy_backend_master_finish_handler: cannot parse results from the master backend
rspamd[8247]: <94608b>; milter; rspamd_milter_send_task_results: cannot find scan results, tempfail
postfix/cleanup[14701]: 4D907F5874: milter-reject: END-OF-MESSAGE from fence.waver.volia.net[93.73.239.187]: 4.7.1 Service unavailable - try again later; from=<guido@bettariniautogru.it> to=<[...]> proto=ESMTP helo=<fence.waver.volia.net>
```

**Comment by fatalbanana** [comment]
Should be fixed in 1.6.6.
